source/_docs/manual-import.md

```diff
@@ -123,7 +123,7 @@ sleep 180
 fi
 done
 ```
-This script connects to your Pantheon site's Dev environment and starts uploading your files. If an error occurs during transfer, rather than stopping, it waits 180 seconds and picks up where it left off.  
+This script connects to your Pantheon site's Dev environment and starts downloading your files. If an error occurs during transfer, rather than stopping, it waits 180 seconds and picks up where it left off.  

```

Comment:
@rachelwhitton I agree that in the context of imports, we should go with uploading. The example script we are showing is actually for downloading. Perhaps we should just change the script example so that it uploads?

`rsync -rlvz --size-only --ipv4 --progress -e 'ssh -p 2222' ./files/* --temp-dir=../tmp/ $ENV.$SITE@appserver.$ENV.$SITE.drush.in:files/`